streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -76,8 +81,9 @@
 
     private final String name;
     private final String parentDir;
+    private final Set<KeyValueIterator> openIterators = new HashSet<>();
 
-    protected File dbDir;
+    File dbDir;
```

Comment:
@guozhangwang, Damian has made the field less accessible than before. Protected allows package access and subclass access while package private does the former only. Since the build is still fine, I'll merge this.